docs/destinations/warehouses-lakes/querying-data/index.md

```diff
@@ -80,12 +80,16 @@ You can query fields in the self-describing event like so:
 ```sql
 SELECT
     ...
-    unstruct_event_my_example_event_1_0_0.my_field,
+    unstruct_event_my_example_event_1.my_field,
     ...
 FROM
     <events>
 ```
 
+:::note
+The column name produced by previous versions of the BigQuery Loader (<2.0.0) would contain full schema version, e.g. `unstruct_event_my_example_event_1_0_0`
```

Comment:
@stanch can I get your opinion -- does this note need amending, now that the v2 loader is also capable of loading the legacy-style columns?  Should we separate out the three different scenarios?

- V2 loader in normal mode
- V2 loader in legacy mode
- V1 loader

....or alternatively hide/ignore the detail that V2 loader has a legacy mode.

Reply:
I think here we should state the V2 normal behavior. And mention that there is a compatibility mode fully described in the migration guide (linking to it).